eng/pipelines/coreclr/templates/build-job.yml

```diff
@@ -141,6 +141,16 @@ jobs:
     - name: SignType
       value: $[ coalesce(variables.OfficialSignType, 'real') ]
 
+    - name: clrRuntimePortableBuildArg
+      value: ''
+    - name: clrManagedPortableBuildArg
+      value: ''
+    - ${{ if eq(parameters.archType, 'armel' )}}:
+      - name: clrRuntimePortableBuildArg
+        value: '-portablebuild=false'
+      - name: clrManagedPortableBuildArg
+        value: '--portablebuild false'
```

Comment:
Could it be just one variable rather than two? iow, is there a scenario where clrRuntimePortableBuildArg != clrManagedPortableBuildArg? In practical sense, I think it will not work because host resolver that uses runtime graph will actually fail if runtime is -portable and libs are non-portable. The opposite can work, but afaict, it is either PortableBuild=true or false for all subsets alike.

Reply:
Thanks for the review.
String value of clrRuntimePortableBuildArg and clrManagedPortableBuildArg are different. "-portablebuild=false" vs "--portablebuild false". I can remove clrManagedPortableBuildArg. And then I checked build and execution on a tizen armel device with the instructions in build-job.yml.
Thank you.

Reply:
Eventually they all get processed by the name handler: https://github.com/dotnet/runtime/blob/2053d653bfd7fc0a5d073a5d31e01600d2d5fa3f/eng/native/build-commons.sh#L410
In build-runtime, it just strips the extra hyphen, but `-portablebuild=false` should work for clr, libs and host subsets at least.

Reply:
```build.sh``` uses ```eng/build.sh```.
In ```eng/build.sh``, portable argument is processed not by ```runtime/eng/native/build-commons.sh```
https://github.com/dotnet/runtime/blob/9d9b465f0be92a0d7584c4b8ad6b873be7f0dbc9/eng/build.sh#L403-L414

And I checked ```build.sh``` with ```-portablebuild=false``` too.
It fails to get a DistroRID what I want and it shows below error message.
```
__DistroRid: linux-armel
MSBUILD : error MSB1001: Unknown switch.
Switch: -portablebuild=false

For switch syntax, type "MSBuild -help"
Build failed with exit code 1. Check errors above.
```

Reply:
Ah right. Currently approach is to use `/p:PortableBuild=false` with build.sh and build-runtime.sh.

We could actually change https://github.com/dotnet/runtime/blob/9d9b465f0be92a0d7584c4b8ad6b873be7f0dbc9/eng/native/build-commons.sh#L299
to
https://github.com/dotnet/runtime/blob/2053d653bfd7fc0a5d073a5d31e01600d2d5fa3f/eng/build.sh#L166
(with` lowerI` as variable name; strips leading hyphen)

Reply:
Right, we can change the build script for better argument handling. However, I think it is better to make a new issue (or new PR) for consistent argument handling in build scripts.
Thank you.